logictest/testdata/aggregate/aggregate_nils

```diff
@@ -0,0 +1,26 @@
+createtable schema=default
+----
+
+insert cols=(labels.label1, labels.label2, stacktrace, timestamp, value)
+value1  null    stack1  1   1
+null    value2  stack1  2   2
+null    value2  stack1  3   3
+----
+
+exec
+select sum(value) as value_sum group by labels.label2
+----
+        1
```

Comment:
Interesting that these blank values make the test pass. I assume they're considered nulls? Do you know why this is passing? In any case, I think it would be good to be explicit about the null value here.

Reply:
i am not sure, initially i tried to use `null` in the result but it ended up failing with this error 

```
logic_test.go:78: 
        testdata/aggregate/aggregate_nulls:10:
         select sum(value) as value_sum group by labels.label2
        expected:
        null    1
        value2  5
        
        found:
                1
        value2  5
```

so i switched to whitespace

Reply:
Hmm, I guess it's probably an artifact of the aggregator that doesn't correctly set nulls in the output (and instead sets an empty string). This is probably something to open an issue about as well although it's not too high priority. We can probably merge this as is with at least a comment pointing to the issue.